examples/plot_firing_pattern.py

```diff
@@ -15,41 +15,64 @@
 # Let us import hnn_core
 
 import hnn_core
-from hnn_core import read_params, Network
-from hnn_core.network_builder import NetworkBuilder
+from hnn_core import read_params, Network, simulate_dipole
 
 hnn_core_root = op.dirname(hnn_core.__file__)
 
 ###############################################################################
-# Then we read the parameters file
+# Then we read the parameters file and enable somatic voltage recording
 params_fname = op.join(hnn_core_root, 'param', 'default.json')
 params = read_params(params_fname)
+params['record_vsoma'] = 1
```

Comment:
this is unpythonic and makes documentation of parameters hard. I want to discourage this pattern in the command line. Can we instead add this parameter to one of the functions like `simulate_dipole` ?